README.md

```diff
@@ -3,17 +3,17 @@
 [![Build Status](https://simple.semaphoreci.com/badges/simple-server/branches/master.svg)](https://simple.semaphoreci.com/projects/simple-server)
 [![Ruby Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://github.com/testdouble/standard)
 
-
 This is the backend for the Simple app to help track hypertensive patients across a population.
 
 ## Table of Contents
 
-* [Development](#development)
-* [Documentation](#documentation)
-* [Deployment](#deployment)
-* [Contributing](#contributing)
+- [Development](#development)
+- [Documentation](#documentation)
+- [Deployment](#deployment)
+- [Contributing](#contributing)
```

Comment:
Looks like you may have some auto format changing markdown styles here?  You can revert all these changes to the README to get the diff smaller and isolated to just your CSV changes.